kokoro/gcp_windows/kokoro_build.bat

```diff
@@ -1,20 +1,43 @@
 :: Code under repo is checked out to %KOKORO_ARTIFACTS_DIR%\github.
 :: The final directory name in this path is determined by the scm name specified
 :: in the job configuration
+@echo off
+
 SET REPO_ROOT=%KOKORO_ARTIFACTS_DIR%\github\orbitprofiler
 
 conan config install %REPO_ROOT%\contrib\conan\configs\windows
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 REM We replace the default remotes by our internal artifactory server
 REM which acts as a secure source for prebuilt dependencies.
-copy /Y %REPO_ROOT%\kokoro\conan\config\remotes.json C:\cygwin64\home\kbuilder\.conan\remotes.json
+copy /Y %REPO_ROOT%\kokoro\conan\config\remotes.json %HOME%\.conan\remotes.json
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+conan config set general.revisions_enabled=True
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+conan config set general.parallel_download=8
 if %errorlevel% neq 0 exit /b %errorlevel%
 
-set CONAN_REVISIONS_ENABLED=1
+:: Building conan
+call conan install -pr msvc2019_release -if %REPO_ROOT%\build\ --build outdated %REPO_ROOT%
+if %errorlevel% neq 0 exit /b %errorlevel%
 
-cd %KOKORO_ARTIFACTS_DIR%\github\orbitprofiler
-call build.bat msvc2019_release_x64
```

Comment:
I think this change requires changing the path in kokoro_build_release.bat as well.